telegram/ext/filters.py

```diff
@@ -131,6 +151,7 @@ def filter(self, message):
             return True
 
     all = _All()
+    """:obj:`Filter`: All Messages."""
```

Comment:
Yes I wrote it this way so users understand this is something you can put in the Filters argument to MessageHandler.